client/my-sites/email/email-providers-stacked-comparison/provider-cards/professional-email-card.tsx

```diff
@@ -1,9 +1,43 @@
+import { TITAN_MAIL_MONTHLY_SLUG } from '@automattic/calypso-products';
 import { Gridicon } from '@automattic/components';
+import formatCurrency from '@automattic/format-currency';
+import { withShoppingCart } from '@automattic/shopping-cart';
+import classNames from 'classnames';
 import { translate } from 'i18n-calypso';
-import React from 'react';
-import poweredByTitanLogo from 'calypso/assets/images/email-providers/titan/powered-by-titan.svg';
-import { getTitanProductName } from 'calypso/lib/titan';
-import type { ProviderCard } from 'calypso/my-sites/email/email-providers-stacked-comparison';
+import page from 'page';
+import React, { FunctionComponent, useState } from 'react';
+import { connect } from 'react-redux';
+import poweredByTitanLogo from 'calypso/assets/images/email-providers/titan/powered-by-titan-stacked.svg';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { fillInSingleCartItemAttributes } from 'calypso/lib/cart-values';
+import { titanMailMonthly } from 'calypso/lib/cart-values/cart-items';
+import {
+	getSelectedDomain,
+	canCurrentUserAddEmail,
+	getCurrentUserCannotAddEmailReason,
+} from 'calypso/lib/domains';
+import { hasEmailForwards } from 'calypso/lib/domains/email-forwarding';
+import { getTitanProductName, isDomainEligibleForTitanFreeTrial } from 'calypso/lib/titan';
+import { TITAN_PROVIDER_NAME } from 'calypso/lib/titan/constants';
+import {
+	areAllMailboxesValid,
+	buildNewTitanMailbox,
+	transformMailboxForCart,
+	validateMailboxes as validateTitanMailboxes,
+} from 'calypso/lib/titan/new-mailbox';
+import withCartKey from 'calypso/my-sites/checkout/with-cart-key';
+import EmailProvidersStackedCard from 'calypso/my-sites/email/email-providers-stacked-comparison/email-provider-stacked-card';
+import {
+	PASSWORD_RESET_TITAN_FIELD,
+	FULL_NAME_TITAN_FIELD,
+} from 'calypso/my-sites/email/titan-new-mailbox';
+import TitanNewMailboxList from 'calypso/my-sites/email/titan-new-mailbox-list';
+import { FullWidthButton } from 'calypso/my-sites/marketplace/components';
+import { getCurrentUserCurrencyCode } from 'calypso/state/currency-code/selectors';
+import { getProductBySlug, getProductsList } from 'calypso/state/products-list/selectors';
+import { getDomainsBySiteId } from 'calypso/state/sites/domains/selectors';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import { EmailProvidersStackedCardProps, ProviderCard } from './provider-card-props';
```

Comment:
Shouldn't these both be types?
```suggestion
import type { EmailProvidersStackedCardProps, ProviderCard } from './provider-card-props';
```